corehq/apps/domain/static/domain/js/case-search-config.js

```diff
@@ -0,0 +1,64 @@
+/* globals hqDefine, ko, $, _ */
+
+hqDefine('domain/js/case-search-config.js', function () {
+    'use strict';
+
+    var module = {};
+
+    var CaseTypeProps = function (caseType, properties) {
+        var self = this;
+        self.case_type = ko.observable(caseType);
+        self.properties = ko.observableArray(properties);
+        // TODO: ^^^ This observableArray isn't changing the values of the strings inside it.
```

Comment:
I don't think you can just change strings in an observableArray. From the knockout docs:
> Key point: An observableArray tracks which objects are in the array, not the state of those objects